src/_guides/language/language-tour.md

```diff
@@ -4544,13 +4544,17 @@ Iterable<int> naturalsDownFrom(int n) sync* {
 }
 ```
 
-
 ## Callable classes
 
 To allow an instance of your Dart class to be called like a function,
 implement the `call()` method.
 
-In the following example, the `WannabeFunction` class defines a call() function
+The `call()` method is a special method
+that allows any class that defines it to emulate a function,
+while supporting the same functionality as normal [functions](#functions)
+such as parameters and return types.
```

Comment:
**issue:** Tighten the language. The `...is a special method that...` does not aid in reader understanding. I would also cut this into shorter sentences.

```suggestion
The `call()` method allows any class that defines it to emulate a function.
This method supports the same functionality as normal [functions](#functions)
such as parameters and return types.
```